docs/methoddocs/high_risk_employee.md

```diff
@@ -0,0 +1,15 @@
+# High Risk Employee
+
+```eval_rst
+.. autoclass:: py42.modules.detectionlists.DetectionListsModule
+    :members:
+    :show-inheritance:
+```
+
+## Detection List
```

Comment:
Should be `Detection Lists` (plural). Also, probably should include a newline under it like on the other one

Reply:
Changed.